django_loci/base/admin.py

```diff
@@ -71,6 +71,12 @@ class Media:
         )
         css = {'all': ('django-loci/css/loci.css',)}
 
+    def clean(self):
+        # if changing existing location to outdoor, delete all floorplans
+        if self.instance and self.cleaned_data.get('type') == "outdoor":
```

Comment:
```suggestion
        if self.instance and self.cleaned_data.get('type') == 'outdoor':
```
Please be consistent with the quoting style of strings.